src/debug-agent.cc

```diff
@@ -299,7 +287,6 @@ void Agent::ChildSignalCb(uv_async_t* signal) {
     if (a->wait_)
```

Comment:
I think this changes semantics. Shouldn't the message be preserved in queue if `a->wait_` is `true`?